### PR TITLE
aws parallel vagrant up: Disable vagrant hostmanager in Vagrantfile, update README for usage

### PR DESCRIPTION
**Pull request by granders**
Parallel cluster bringup on aws! (sortof)

The instructions were arrived at through trial and error, but seem to work fairly consistently.

The previous source of badness in parallel bringup seems to have been the *interaction* between `vagrant-aws` and `vagrant-hostmanager`.

### CODE REVIEW

**Comment by granders** [comment]
Ping @ewencp

**Comment by granders** [comment]
Also, fyi - I actually follow this procedure in the Jenkins muckrake 2.x job, and it works consistently

File: README.md [hunk 1]
```diff
@@ -140,7 +145,10 @@ EC2 Quickstart
 
 * Start up the instances:
 
-        vagrant up --provider=aws --no-provision --no-parallel && vagrant provision
+        # Start up machines in batches of size <= 8
+        vagrant up --provider=aws worker1 worker2 worker3 worker4 worker5 worker6 worker7 worker8
+        vagrant up --provider=aws worker9 worker10
+        vagrant hostmanager
```
Comment (ewencp) [inline README.md]: Should we maintain the `--no-parallel` version as an alternative? It is slower, but also simpler and a lot less awkward than this version.

**Comment by ewencp** [comment]
@granders Minor comment re: instructions, but LGTM

**Comment by granders** [comment]
@ewencp I have actually now added a wrapper script around `vagrant up` which is worth taking a look at.

I've used this script successfully with a cluster of size 30, but I think the actually AWS rate limiting threshold may vary with time (due to load?), so the `--max-parallel` setting may need to be tweaked by users if they see rate limit errors.

I set a conservative value of `5` as the default for max parallel which should hopefully give enough headroom for most to be able to ignore this setting altogether on a day to day basis.

**Pull request merged**